_posts/2024-12-12-WildFly35Beta-released.adoc

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title:  "New WildFly 35 Beta release"
+date:   2024-12-12
+tags:   announcement release
+author: darranl
+description: WildFly 35 Beta is now available for download.
+---
+
+= WildFly 35 Beta is released!
+
+I'm pleased to announce that the new WildFly 35.0.0.Beta1 release is available for download at https://wildfly.org/downloads.
+
+[[new_and_notable]]
+== New and Notable
+
+This quarter we had a heavy focus on MicroProfile, particularly link:https://microprofile.io/2024/08/22/microprofile-7-0-release/#[MicroProfile 7.0, window=_blank].
+
+* WildFly now link:https://docs.wildfly.org/wildfly-proposals/microprofile/WFLY-19592_MicroProfile_Fault_Tolerance_4_1.html[supports MicroProfile Fault Tolerance 4.1, window=_blank].
+
+* WildFly now link:https://docs.wildfly.org/wildfly-proposals/microprofile/WFLY-19591_MicroProfile_OpenAPI_4.0.html[supports MicroProfile OpenAPI 4.0, window=_blank].
+
+* Standard WildFly now link:https://docs.wildfly.org/wildfly-proposals/microprofile/WFLY-19866-promote-mp-rest-client-to-default.html[supports MicroProfile REST Client 4.0, window=_blank]. This was previously supported in WildFly Preview.
+
+* Standard WildFly now link:https://docs.wildfly.org/wildfly-proposals/observability/WFLY-19846-promote-mp-tel-2-to-default.html[supports MicroProfile Telemetry 2.0, window=_blank]. This was previously supported in WildFly Preview.
+
+* Our MicroProfile Reactive Messaging subsystem has added OpenTelemetry tracing integration for link:https://docs.wildfly.org/wildfly-proposals/microprofile/WFLY-19835_microprofile_reactive_messaging_otel_with_kafka.html[Kafka connectors, window=_blank] and link:https://docs.wildfly.org/wildfly-proposals/microprofile/WFLY-19836_microprofile_reactive_messaging_otel_with_amqp.html[AMQP connectors, window=_blank].
```

Comment:
Nothing you need to deal with, but these two pages don't render properly (no title). I thought I fixed them last night but it seems not. I'll try again.

Reply:
I fixed the rendering on these, and posted on wildfly-dev about what the problem was.